app/services/statement_classifier.rb

```diff
@@ -74,7 +79,11 @@ def classified_statements
   end
 
   def statement_type(statement)
-    if statement.unverifiable?
+    if statement.removed_from_source? && !statement.done?
+      nil
```

Comment:
This now introduces the possibility that the statement type can be nil, which it couldn't be before, so I think there'll need to be some frontend change to handle when `currentView` returns `null` instead of a component. (Or maybe I've missed that...) Or just ignoring any statements with null type when loading, I suppose.

Reply:
Yes. Good catch, I had thought these were getting filtered out due to the `#classified_statements` method but that isn't being used. I have updated the ruby code to make `#to_a` filter statements without a type.

Reply:
See dab9c5699effbc1d718cdc2f2ff19374c6fd82e3